src/Console/ConfigurationResolver.php

```diff
@@ -314,13 +314,15 @@ private function computeConfigFiles()
         }
 
         $path = $this->path;
-        if (is_file($path) && $dirName = pathinfo($path, PATHINFO_DIRNAME)) {
-            $configDir = $dirName;
-        } elseif ($this->isStdIn || null === $path) {
+
+        if ($this->isStdIn || 0 === count($path)) {
             $configDir = $this->cwd;
-            // path is directory
+        } elseif (1 < count($path)) {
+            throw new InvalidConfigurationException('For multiple paths config parameter is required.');
```

Comment:
It was a design decision.
Basically, if you provide a path but don't provide a config file, it's localization is automatically discovered. if you provide multiple path, autodiscovering is extra tricky and could lead to discover multiple config files, while tool can only use one config per run.